ontotext-yasgui-web-component/src/services/yasgui/configuration/yasgui-configuration-builder.ts

```diff
@@ -102,6 +107,27 @@ export class YasguiConfigurationBuilder {
     return config;
   }
 
+  private initShortcuts(config: YasguiConfiguration): void {
+    const keyboardShortcutDescriptions = KeyboardShortcutService.initKeyboardShortcutMapping();
+    const extraKeys = {};
+    keyboardShortcutDescriptions.forEach(keyboardShortcutDescription => {
+      keyboardShortcutDescription.keyboardShortcuts.forEach(keyboardShortcut => {
+        extraKeys[keyboardShortcut] = keyboardShortcutDescription.executeFunction;
+      });
+    });
+    config.yasguiConfig.yasqe.extraKeys = extraKeys;
+
+    config.yasguiConfig.yasqe.keyboardShortcutDescriptions = keyboardShortcutDescriptions.map(keyboardShortcutDescription => keyboardShortcutDescription.NAME.toString());
```

Comment:
Maybe use toLowerCase or even change the enum KeyboardShortcutName values to be lowercase